include/libzfs.h

```diff
@@ -291,7 +291,8 @@ typedef struct trimflags {
 /*
  * Functions to manipulate pool and vdev state
  */
-_LIBZFS_H int zpool_scan(zpool_handle_t *, pool_scan_func_t, pool_scrub_cmd_t);
+_LIBZFS_H int zpool_scan(zpool_handle_t *, pool_scan_func_t, pool_scrub_cmd_t,
+    time_t, time_t);
```

Comment:
If we could add a new function and avoid modifying this public function it looks like we could avoid a major version number bump to the libzfs library.